src/features/domain-settings/DomainSettingsHeader.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+
+import { FormStep, steps } from '.';
+import classNames from 'classnames/bind';
+
+import { IconXClose } from '@zero-tech/zui/components/Icons';
+import { Wizard, StepBar, Step } from '@zero-tech/zui/components';
+
+import styles from './DomainSettingsHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+interface DomainSettingsHeaderProps {
+	zna: string;
+	stepId: string;
+	headerText: string;
+	isTransactionLoading: boolean;
+	onClose: () => void;
+	onStepBarUpdate: (step: Step) => void;
```

Comment:
nit: would this function name make more sense as `onStepBarNavigation`? `onStepBarUpdate` is a little bit ambiguous.